src/docs/arch-compact.dox

```diff
@@ -62,4 +69,21 @@ track current progress. Some of the verbose logs for compaction can be activated
 verbose logs, WT also records statistics about compact progress that can be used to debug issues
 with the compaction process.
 
+# Background compaction #
+
+Background compaction is a WiredTiger service that runs compaction autonomously on all the eligible files
+present in the database while being the least disruptive as possible. Every file is eligible unless:
+
+    - it belongs to a tiered table or,
+    - it is part of the exclude list that can be configured when enabling background compaction.
```

Comment:
"it is a part of" (you're missing an article)

Reply:
Fixed in fe7a5bfc6aa1b1fbb3fd94285251c87a3b3361a5, thank you!